### PR TITLE
Merge all misc commands into single CLI

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: dataplane/luciusctl/luciusctl.go [hunk 1]
```diff
@@ -12,21 +12,20 @@
...
 	"github.com/openconfig/lemming/dataplane/luciusctl/info"
 	"github.com/openconfig/lemming/dataplane/luciusctl/sai"
 )
 
-func main() {
-	cmd := cobra.Command{
-		Use: "luciusctl",
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "lucius",
```
Comment (alexmasi) [inline dataplane/luciusctl/luciusctl.go:27]: should this still be luciusctl?

**Review by alexmasi — APPROVED** [review]
any backwards compatibility concerns?

File: dataplane/luciusctl/luciusctl.go (hunk 1 above)
Comment (DanG100) [reply dataplane/luciusctl/luciusctl.go:27]: i thought that was too many characters to type

**Comment by DanG100** [comment]
> any backwards compatibility concerns?

i don't think any one other than me is using these tools to debug

**Pull request merged**